numba/core/interpreter.py

```diff
@@ -1818,6 +1818,8 @@ def op_LIST_EXTEND(self, inst, target, value, extendvar, res):
                 else:
                     ok = False
                     break
+        if ok and build_empty_list is None:
+            raise errors.UnsupportedError("SPECIALSTRING")
```

Comment:
Change the err msg to something more informative

Reply:
done in 8c7a69d804b9213e2a46c28eccede4eb930d437b

Reply:
now as a5497934cc3faa8d46e12b97d59f166faf5e4161